capsules/calculator_object_speed/stream_state.py

```diff
@@ -0,0 +1,54 @@
+from collections import OrderedDict
+from typing import Optional, Tuple
+from uuid import UUID
+from dataclasses import dataclass
+
+from vcap import BaseStreamState
+
+
+@dataclass
+class DetectionTimestamp:
+    tstamp: float
+    """The timestamp when the detection was seen"""
+
+    coordinate: Tuple[int, int]
+    """The coordinate where the object was seen at that timestamp. This 
+    represents the centroid of the bounding box of the detection. 
+    """
+
+
+class StreamState(BaseStreamState):
+    """Basically an LRU cache that stores the last-seen timestamp and
+    coordinate keyed to detection.track_id
+    """
+
+    def __init__(self):
+        self.cache = OrderedDict()
+        """How many detection's information it can keep in the cache"""
+
+    def get(self, key: UUID) -> Optional[DetectionTimestamp]:
+        """
+        :param key: The detection.track_id
+        :return: (timestamp, (x, y)) or (None, None) if there was no detection
+            information for this track ID
```

Comment:
It looks like you forgot to update the docstring here when you created the new type

Reply:
Fixed